src/JsonSchema/TypeFactory.php

```diff
@@ -17,6 +17,12 @@
 use ApiPlatform\Core\Util\ResourceClassInfoTrait;
 use Ramsey\Uuid\UuidInterface;
 use Symfony\Component\PropertyInfo\Type;
+use function array_key_exists;
+use function array_merge;
+use function array_unique;
+use function array_values;
+use function is_array;
+use function is_string;
```

Comment:
Why?

Reply:
Mostly something I do by default: not sure if there's a coding standard for this project, didn't check build failures on that front yet.

Overall, relying on global function fallback is more trouble.